Finger-tip collision elements for wsg model.

avalenzu opened the pull request:
This PR replaces the line of spheres along the WSG fingers with a triangle of spheres at each finger tip. The result looks like this:
![screenshot-2017-10-26_13 58 52](https://user-images.githubusercontent.com/2574034/32068993-daf30da8-ba55-11e7-9555-be0a8bd34561.png)
 I lowered the alpha on the left finger for this screenshot so that all three spheres are visible.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/robotlocomotion/drake/7345)
<!-- Reviewable:end -->

avalenzu commented:
+@naveenoid for feature review.

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-:-KxOq8I1s0Ngv6G6G_mN:b-ijjxgm)*
<!-- Sent from Reviewable.io -->

SeanCurtis-TRI commented:
I note that the collision elements are not bound by the visual geometry. Is that what you meant? If the visual geometry represents the physical extent of the actual gripper, then you might want to embed all of the contact geometry inside the physical scope. As it is, the gripper will interact with objects *before* the physical object would.

naveenoid commented:
+1 to @seancurtis-tri comments. Aside from that, only a BTW request.

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_ball_contact.sdf, line 3 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-KxP23BQ8btUxvsfGBCk:-KxP23BQ8btUxvsfGBCl:b6wg7wj) ([raw file](https://github.com/robotlocomotion/drake/blob/396913115e66e146a973975630e070d38010ae35/drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_ball_contact.sdf#L3)):*
> ```
> <?xml version='1.0'?>
> <sdf version='1.6'>
>   <model name='Schunk_Gripper'>
> ```

BTW: Consider adding a general comment here mentioning the specific contact elements present in this model and its need perhaps.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-:-KxP2FNVnE3uqmFZm76O:b60clb3)*
<!-- Sent from Reviewable.io -->

avalenzu commented:
Agreed. I've modified the collision geometries to address these concerns and the CI failure. The result is shown below:

![Screenshot-2017-10-26\_16:10:16.png](https://files.reviewableusercontent.io/images/reviewable/2574034/2BSKCarRd-XpxchNBU3DGgYA/Screenshot-2017-10-26_16%3A10%3A16.png) 

I've removed the visual elements for the spheres from the SDF-file in the PR, since
 1. They would always be covered by the box
 2. They were not present in the original SDF

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_ball_contact.sdf, line 3 at r1](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-KxP23BQ8btUxvsfGBCk:-KxPLRM4JUjovJIh5Xd6:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/396913115e66e146a973975630e070d38010ae35/drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_ball_contact.sdf#L3)):*
<details><summary><i>Previously, naveenoid (Naveen Kuppuswamy) wrote…</i></summary><blockquote>

BTW: Consider adding a general comment here mentioning the specific contact elements present in this model and its need perhaps.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-:-KxPJiWxOdxtF90r4RN0:b-tanap)*
<!-- Sent from Reviewable.io -->

naveenoid commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-:-KxTQCMm4jO7l5xBG-_K:bnfp4nl)*
<!-- Sent from Reviewable.io -->

avalenzu commented:
+@sherm1  for platform review. 

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-:-KxTUMIqRrbn2YVWZPC1:boner6o)*
<!-- Sent from Reviewable.io -->

sherm1 commented:
Platform <img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>. Love the pictures in the PR!

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/7345#-:-KxUfKt7qaDEOSis2e8G:beeum15)*
<!-- Sent from Reviewable.io -->
